React/books/src/app/components/Book/index.js

```diff
@@ -5,34 +5,48 @@ import PropTypes from 'prop-types';
 import actionsCreators from '../../../redux/Book/actions';
 
 class Book extends Component {
-  constructor(props) {
-    super(props);
-
-    setTimeout(() => {
-      this.getBooks();
-    }, 2000);
-  }
+  componentDidMount = () => {
+    this.getBooks();
+  };
 
   getBooks = () => {
     this.props.dispatch(actionsCreators.getBooks());
   };
 
+  getUlListBooks = () => {
+    if (this.props.listBooks) {
+      return this.props.listBooks.map(book => <li key={book.id}>{`${book.title} ${book.author}`}</li>);
+    }
+  };
+
+  getStateBooks() {
+    if (this.props.loading) {
+      return 'Cargando...';
+    }
+    return 'Books:';
+  }
+
   render() {
     return (
       <div>
-        <p>{this.props.text}</p>
-        <button onClick={this.getBooks}>RECARGAR</button>
+        <h2>{this.getStateBooks()}</h2>
+        <ul>{this.getUlListBooks()}</ul>
```

Comment:
We usually call this methods `renderSomething()`.
Also you can do it inline like this:
```js
<h2>{this.props.loading ? 'Cargando...' : 'Books:'}</h2>
```